examples/employee-app/pom.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+
+    Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+        <artifactId>helidon-examples-project</artifactId>
+        <groupId>io.helidon.examples</groupId>
+        <version>1.0.0</version>
+    </parent>
+	
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.example.helidon.employee</groupId>
+    <artifactId>employee-app</artifactId>
+    <packaging>jar</packaging>
+    <name>${project.artifactId}</name>
+
+    <properties>
+        <!-- Default package. Will be overriden by Maven archetype -->
+        <package>com.example.helidon.employee</package>
+        <mainClass>com.example.helidon.employee.Main</mainClass>
+        <maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
+        <libs.classpath.prefix>libs</libs.classpath.prefix>
+        <copied.libs.dir>${project.build.directory}/${libs.classpath.prefix}</copied.libs.dir>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <argLine>-Dfile.encoding=UTF-8</argLine>
+    </properties>
+
+    <build>
+        <finalName>${project.artifactId}</finalName>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>3.1</version>

```

Comment:
No need to repeat the plugin versions, they are inherited from the parent pom.

Reply:
This should be fixed now.

Reply:
I still see versions for all plugins.

Reply:
Sorry. This should be fixed now.

Reply:
I should I've been more clear, since the plugin versions are already managed, you should ommit pluginManagement entries that only define a version. I.e, in your case you'd only keep the pluginManagement entry for maven-jar-plugin without version.